Build error after recent commits

Good afternoon!

I'm facing a build error after yesterdays changes, I've not bisected but presumably the culrpit is 3a96516686073d479f67fef5593c225c3c363b64 judging by the error message:
```llvm
ninja: job failed: /home/ray/mingw-test/build/install/bin/x86_64-w64-mingw32-clang -Iq2pro.exe.p -I. -I.. -I../inc -I../subprojects/libvorbis-1.3.7/include -Isubprojects/khr-headers-20220530 -I../subprojects/khr-headers-20220530 -Isrc/windows -I/home/ray/mingw-test/build/install/mingw/include -I/home/ray/mingw-test/build/install/mingw/include/libpng16 -I/home/ray/mingw-test/build/install/mingw/include/SDL2 -I/home/ray/mingw-test/build/install/mingw/include/AL -fvisibility=hidden -fcolor-diagnostics -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu99 -O3 -DHAVE_CONFIG_H -DUSE_PROTOCOL_EXTENSIONS=1 -fsigned-char -fms-extensions -fno-math-errno -Wno-microsoft-anon-tag -Wno-tautological-constant-out-of-range-compare -Wpointer-arith -Wformat-security -Werror=vla -Wstrict-prototypes -D__USE_MINGW_ANSI_STDIO=1 -march=x86-64-v3 -faddrsig -fuse-ld=lld -mllvm -polly -mllvm -polly-vectorizer=stripmine -mguard=cf -fcf-protection=full -DAL_LIBTYPE_STATIC -Dmain=SDL_main -DCURL_STATICLIB -DUSE_CLIENT=1 -DUSE_REF=1 -D_WIN32_WINNT=0x0601 -Wmissing-prototypes -MD -MQ q2pro.exe.p/src_common_bsp.c.obj -MF q2pro.exe.p/src_common_bsp.c.obj.d -o q2pro.exe.p/src_common_bsp.c.obj -c ../src/common/bsp.c
../src/common/bsp.c:974:56: error: call to undeclared function 'Com_MakePrintable'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  974 |             Com_WPrintf("Bad material \"%s\" in %s\n", Com_MakePrintable(out->material), path);
```

Full Build log:
```llvm
>> ninja -j1
[1/220] Compiling C object gamex86_64.dll.p/src_shared_shared.c.obj
[2/220] Compiling C object gamex86_64.dll.p/src_shared_m_flash.c.obj
[3/220] Compiling C object gamex86_64.dll.p/src_game_p_weapon.c.obj
[4/220] Compiling C object gamex86_64.dll.p/src_game_p_view.c.obj
[5/220] Compiling C object gamex86_64.dll.p/src_game_p_trail.c.obj
[6/220] Compiling C object gamex86_64.dll.p/src_game_p_hud.c.obj
[7/220] Compiling C object gamex86_64.dll.p/src_game_p_client.c.obj
[8/220] Compiling C object gamex86_64.dll.p/src_game_m_tank.c.obj
[9/220] Compiling C object gamex86_64.dll.p/src_game_m_supertank.c.obj
[10/220] Compiling C object gamex86_64.dll.p/src_game_m_soldier.c.obj
[11/220] Compiling C object gamex86_64.dll.p/src_game_m_parasite.c.obj
[12/220] Compiling C object gamex86_64.dll.p/src_game_m_mutant.c.obj
[13/220] Compiling C object gamex86_64.dll.p/src_game_m_move.c.obj
[14/220] Compiling C object gamex86_64.dll.p/src_game_m_medic.c.obj
[15/220] Compiling C object gamex86_64.dll.p/src_game_m_insane.c.obj
[16/220] Compiling C object gamex86_64.dll.p/src_game_m_infantry.c.obj
[17/220] Compiling C object gamex86_64.dll.p/src_game_m_hover.c.obj
[18/220] Compiling C object gamex86_64.dll.p/src_game_m_gunner.c.obj
[19/220] Compiling C object gamex86_64.dll.p/src_game_m_gladiator.c.obj
[20/220] Compiling C object gamex86_64.dll.p/src_game_m_flyer.c.obj
[21/220] Compiling C object gamex86_64.dll.p/src_game_m_float.c.obj
[22/220] Compiling C object gamex86_64.dll.p/src_game_m_flipper.c.obj
[23/220] Compiling C object gamex86_64.dll.p/src_game_m_chick.c.obj
[24/220] Compiling C object gamex86_64.dll.p/src_game_m_brain.c.obj
[25/220] Compiling C object gamex86_64.dll.p/src_game_m_boss3.c.obj
[26/220] Compiling C object gamex86_64.dll.p/src_game_m_boss32.c.obj
[27/220] Compiling C object gamex86_64.dll.p/src_game_m_boss31.c.obj
[28/220] Compiling C object gamex86_64.dll.p/src_game_m_boss2.c.obj
[29/220] Compiling C object gamex86_64.dll.p/src_game_m_berserk.c.obj
[30/220] Compiling C object gamex86_64.dll.p/src_game_m_actor.c.obj
[31/220] Compiling C object gamex86_64.dll.p/src_game_g_weapon.c.obj
[33/220] Compiling C object gamex86_64.dll.p/src_game_g_turret.c.obj
[34/220] Compiling C object gamex86_64.dll.p/src_game_g_trigger.c.obj
[35/220] Compiling C object gamex86_64.dll.p/src_game_g_target.c.obj
[36/220] Compiling C object gamex86_64.dll.p/src_game_g_svcmds.c.obj
[37/220] Compiling C object gamex86_64.dll.p/src_game_g_spawn.c.obj
[38/220] Compiling C object gamex86_64.dll.p/src_game_g_save.c.obj
[39/220] Compiling C object gamex86_64.dll.p/src_game_g_ptrs.c.obj
[40/220] Compiling C object gamex86_64.dll.p/src_game_g_phys.c.obj
[41/220] Compiling C object gamex86_64.dll.p/src_game_g_monster.c.obj
[42/220] Compiling C object gamex86_64.dll.p/src_game_g_misc.c.obj
[43/220] Compiling C object gamex86_64.dll.p/src_game_g_main.c.obj
[44/220] Compiling C object gamex86_64.dll.p/src_game_g_items.c.obj
[45/220] Compiling C object gamex86_64.dll.p/src_game_g_func.c.obj
[46/220] Compiling C object gamex86_64.dll.p/src_game_g_combat.c.obj
[47/220] Compiling C object gamex86_64.dll.p/src_game_g_cmds.c.obj
[48/220] Compiling C object gamex86_64.dll.p/src_game_g_chase.c.obj
[49/220] Compiling C object gamex86_64.dll.p/src_game_g_ai.c.obj
[50/220] Generating src/windows/res_game.rc with a custom command
[51/220] Linking target gamex86_64.dll
[52/220] Compiling C object q2proded.exe.p/src_server_world.c.obj
[53/220] Compiling C object q2proded.exe.p/src_server_user.c.obj
[54/220] Compiling C object q2proded.exe.p/src_server_send.c.obj
[55/220] Compiling C object q2proded.exe.p/src_server_main.c.obj
[56/220] Compiling C object q2proded.exe.p/src_server_init.c.obj
[57/220] Compiling C object q2proded.exe.p/src_server_game.c.obj
[58/220] Compiling C object q2proded.exe.p/src_server_entities.c.obj
[59/220] Compiling C object q2proded.exe.p/src_server_commands.c.obj
[60/220] Compiling C object q2proded.exe.p/src_client_null.c.obj
[61/220] Compiling C object q2proded.exe.p/src_server_save.c.obj
[62/220] Compiling C object q2proded.exe.p/src_server_mvd_parse.c.obj
[63/220] Compiling C object q2proded.exe.p/src_server_mvd_game.c.obj
[64/220] Compiling C object q2proded.exe.p/src_server_mvd_client.c.obj
[65/220] Compiling C object q2proded.exe.p/src_server_mvd.c.obj
[66/220] Compiling C object q2proded.exe.p/src_windows_debug.c.obj
[67/220] Compiling C object q2proded.exe.p/src_windows_hunk.c.obj
[68/220] Compiling C object q2proded.exe.p/src_windows_system.c.obj
[69/220] Compiling C object q2proded.exe.p/src_shared_shared.c.obj
[70/220] Compiling C object q2proded.exe.p/src_common_zone.c.obj
[71/220] Compiling C object q2proded.exe.p/src_common_utils.c.obj
[72/220] Compiling C object q2proded.exe.p/src_common_sizebuf.c.obj
[73/220] Compiling C object q2proded.exe.p/src_common_prompt.c.obj
[74/220] Compiling C object q2proded.exe.p/src_common_pmove.c.obj
[75/220] Compiling C object q2proded.exe.p/src_common_net_net.c.obj
[76/220] Compiling C object q2proded.exe.p/src_common_net_chan.c.obj
[77/220] Compiling C object q2proded.exe.p/src_common_msg.c.obj
[78/220] Compiling C object q2proded.exe.p/src_common_mdfour.c.obj
[79/220] Compiling C object q2proded.exe.p/src_common_math.c.obj
[80/220] Compiling C object q2proded.exe.p/src_common_hash_map.c.obj
[81/220] Compiling C object q2proded.exe.p/src_common_files.c.obj
[82/220] Compiling C object q2proded.exe.p/src_common_fifo.c.obj
[83/220] Compiling C object q2proded.exe.p/src_common_field.c.obj
[84/220] Compiling C object q2proded.exe.p/src_common_error.c.obj
[85/220] Compiling C object q2proded.exe.p/src_common_cvar.c.obj
[86/220] Compiling C object q2proded.exe.p/src_common_common.c.obj
[87/220] Compiling C object q2proded.exe.p/src_common_cmodel.c.obj
[88/220] Compiling C object q2proded.exe.p/src_common_cmd.c.obj
[89/220] Compiling C object q2proded.exe.p/src_common_bsp.c.obj
[90/220] Generating src/windows/res_q2proded.rc with a custom command
[91/220] Linking target q2proded.exe
[92/220] Compiling C object q2pro.exe.p/src_refresh_world.c.obj
[93/220] Compiling C object q2pro.exe.p/src_refresh_texture.c.obj
[94/220] Compiling C object q2pro.exe.p/src_refresh_tess.c.obj
[95/220] Compiling C object q2pro.exe.p/src_refresh_surf.c.obj
[96/220] Compiling C object q2pro.exe.p/src_refresh_state.c.obj
[97/220] Compiling C object q2pro.exe.p/src_refresh_sky.c.obj
[98/220] Compiling C object q2pro.exe.p/src_refresh_shader.c.obj
[99/220] Compiling C object q2pro.exe.p/src_refresh_qgl.c.obj
[100/220] Compiling C object q2pro.exe.p/src_refresh_models.c.obj
[101/220] Compiling C object q2pro.exe.p/src_refresh_mesh.c.obj
[102/220] Compiling C object q2pro.exe.p/src_refresh_main.c.obj
[103/220] Compiling C object q2pro.exe.p/src_refresh_legacy.c.obj
[104/220] Compiling C object q2pro.exe.p/src_refresh_images.c.obj
[105/220] Compiling C object q2pro.exe.p/src_refresh_hq2x.c.obj
[106/220] Compiling C object q2pro.exe.p/src_refresh_draw.c.obj
[107/220] Compiling C object q2pro.exe.p/src_unix_sound_sdl.c.obj
[108/220] Compiling C object q2pro.exe.p/src_client_sound_dma.c.obj
[109/220] Compiling C object q2pro.exe.p/src_client_ui_ui.c.obj
[110/220] Compiling C object q2pro.exe.p/src_client_ui_servers.c.obj
[111/220] Compiling C object q2pro.exe.p/src_client_ui_script.c.obj
[112/220] Compiling C object q2pro.exe.p/src_client_ui_playermodels.c.obj
[113/220] Compiling C object q2pro.exe.p/src_client_ui_playerconfig.c.obj
[114/220] Compiling C object q2pro.exe.p/src_client_ui_menu.c.obj
[115/220] Compiling C object q2pro.exe.p/src_client_ui_demos.c.obj
[116/220] Compiling C object q2pro.exe.p/src_windows_egl.c.obj
[117/220] Compiling C object q2pro.exe.p/src_windows_dsound.c.obj
[118/220] Compiling C object q2pro.exe.p/src_windows_wave.c.obj
[119/220] Compiling C object q2pro.exe.p/src_windows_pthread.c.obj
[120/220] Compiling C object q2pro.exe.p/src_windows_wgl.c.obj
[121/220] Compiling C object q2pro.exe.p/src_windows_client.c.obj
[122/220] Compiling C object q2pro.exe.p/src_client_sound_ogg.c.obj
[123/220] Compiling C object q2pro.exe.p/src_client_sound_qal.c.obj
[124/220] Compiling C object q2pro.exe.p/src_client_sound_al.c.obj
[125/220] Compiling C object q2pro.exe.p/src_unix_video_sdl.c.obj
[126/220] Compiling C object q2pro.exe.p/src_client_http.c.obj
[127/220] Compiling C object q2pro.exe.p/src_shared_m_flash.c.obj
[128/220] Compiling C object q2pro.exe.p/src_server_world.c.obj
[129/220] Compiling C object q2pro.exe.p/src_server_user.c.obj
[130/220] Compiling C object q2pro.exe.p/src_server_send.c.obj
[131/220] Compiling C object q2pro.exe.p/src_server_main.c.obj
[132/220] Compiling C object q2pro.exe.p/src_server_init.c.obj
[133/220] Compiling C object q2pro.exe.p/src_server_game.c.obj
[134/220] Compiling C object q2pro.exe.p/src_server_entities.c.obj
[135/220] Compiling C object q2pro.exe.p/src_server_commands.c.obj
[136/220] Compiling C object q2pro.exe.p/src_common_async.c.obj
[137/220] Compiling C object q2pro.exe.p/src_client_view.c.obj
[138/220] Compiling C object q2pro.exe.p/src_client_tent.c.obj
[139/220] Compiling C object q2pro.exe.p/src_client_sound_mem.c.obj
[140/220] Compiling C object q2pro.exe.p/src_client_sound_main.c.obj
[141/220] Compiling C object q2pro.exe.p/src_client_screen.c.obj
[142/220] Compiling C object q2pro.exe.p/src_client_refresh.c.obj
[143/220] Compiling C object q2pro.exe.p/src_client_predict.c.obj
[144/220] Compiling C object q2pro.exe.p/src_client_precache.c.obj
[145/220] Compiling C object q2pro.exe.p/src_client_parse.c.obj
[146/220] Compiling C object q2pro.exe.p/src_client_newfx.c.obj
[147/220] Compiling C object q2pro.exe.p/src_client_main.c.obj
../src/client/main.c:1755:14: warning: variable 'count' set but not used [-Wunused-but-set-variable]
 1755 |     int len, count, line;
      |              ^
1 warning generated.
[148/220] Compiling C object q2pro.exe.p/src_client_locs.c.obj
[149/220] Compiling C object q2pro.exe.p/src_client_keys.c.obj
[150/220] Compiling C object q2pro.exe.p/src_client_input.c.obj
../src/client/input.c:963:9: warning: variable 'totalCmds' set but not used [-Wunused-but-set-variable]
  963 |     int totalCmds, totalMsec;
      |         ^
../src/client/input.c:963:20: warning: variable 'totalMsec' set but not used [-Wunused-but-set-variable]
  963 |     int totalCmds, totalMsec;
      |                    ^
2 warnings generated.
[151/220] Compiling C object q2pro.exe.p/src_client_entities.c.obj
[152/220] Compiling C object q2pro.exe.p/src_client_effects.c.obj
[153/220] Compiling C object q2pro.exe.p/src_client_download.c.obj
[154/220] Compiling C object q2pro.exe.p/src_client_demo.c.obj
[155/220] Compiling C object q2pro.exe.p/src_client_crc.c.obj
[156/220] Compiling C object q2pro.exe.p/src_client_console.c.obj
[157/220] Compiling C object q2pro.exe.p/src_client_cin.c.obj
[158/220] Compiling C object q2pro.exe.p/src_client_ascii.c.obj
[159/220] Compiling C object q2pro.exe.p/src_server_save.c.obj
[160/220] Compiling C object q2pro.exe.p/src_server_mvd_parse.c.obj
[161/220] Compiling C object q2pro.exe.p/src_server_mvd_game.c.obj
[162/220] Compiling C object q2pro.exe.p/src_server_mvd_client.c.obj
[163/220] Compiling C object q2pro.exe.p/src_server_mvd.c.obj
[164/220] Compiling C object q2pro.exe.p/src_windows_debug.c.obj
[165/220] Compiling C object q2pro.exe.p/src_windows_hunk.c.obj
[166/220] Compiling C object q2pro.exe.p/src_windows_system.c.obj
[167/220] Compiling C object q2pro.exe.p/src_shared_shared.c.obj
[168/220] Compiling C object q2pro.exe.p/src_common_zone.c.obj
[169/220] Compiling C object q2pro.exe.p/src_common_utils.c.obj
[170/220] Compiling C object q2pro.exe.p/src_common_sizebuf.c.obj
[171/220] Compiling C object q2pro.exe.p/src_common_prompt.c.obj
[172/220] Compiling C object q2pro.exe.p/src_common_pmove.c.obj
[173/220] Compiling C object q2pro.exe.p/src_common_net_net.c.obj
[174/220] Compiling C object q2pro.exe.p/src_common_net_chan.c.obj
[175/220] Compiling C object q2pro.exe.p/src_common_msg.c.obj
[176/220] Compiling C object q2pro.exe.p/src_common_mdfour.c.obj
[177/220] Compiling C object q2pro.exe.p/src_common_math.c.obj
[178/220] Compiling C object q2pro.exe.p/src_common_hash_map.c.obj
[179/220] Compiling C object q2pro.exe.p/src_common_files.c.obj
[180/220] Compiling C object q2pro.exe.p/src_common_fifo.c.obj
[181/220] Compiling C object q2pro.exe.p/src_common_field.c.obj
[182/220] Compiling C object q2pro.exe.p/src_common_error.c.obj
[183/220] Compiling C object q2pro.exe.p/src_common_cvar.c.obj
[184/220] Compiling C object q2pro.exe.p/src_common_common.c.obj
[185/220] Compiling C object q2pro.exe.p/src_common_cmodel.c.obj
[186/220] Compiling C object q2pro.exe.p/src_common_cmd.c.obj
[187/220] Compiling C object q2pro.exe.p/src_common_bsp.c.obj
ninja: job failed: /home/ray/mingw-test/build/install/bin/x86_64-w64-mingw32-clang -Iq2pro.exe.p -I. -I.. -I../inc -I../subprojects/libvorbis-1.3.7/include -Isubprojects/khr-headers-20220530 -I../subprojects/khr-headers-20220530 -Isrc/windows -I/home/ray/mingw-test/build/install/mingw/include -I/home/ray/mingw-test/build/install/mingw/include/libpng16 -I/home/ray/mingw-test/build/install/mingw/include/SDL2 -I/home/ray/mingw-test/build/install/mingw/include/AL -fvisibility=hidden -fcolor-diagnostics -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu99 -O3 -DHAVE_CONFIG_H -DUSE_PROTOCOL_EXTENSIONS=1 -fsigned-char -fms-extensions -fno-math-errno -Wno-microsoft-anon-tag -Wno-tautological-constant-out-of-range-compare -Wpointer-arith -Wformat-security -Werror=vla -Wstrict-prototypes -D__USE_MINGW_ANSI_STDIO=1 -march=x86-64-v3 -faddrsig -fuse-ld=lld -mllvm -polly -mllvm -polly-vectorizer=stripmine -mguard=cf -fcf-protection=full -DAL_LIBTYPE_STATIC -Dmain=SDL_main -DCURL_STATICLIB -DUSE_CLIENT=1 -DUSE_REF=1 -D_WIN32_WINNT=0x0601 -Wmissing-prototypes -MD -MQ q2pro.exe.p/src_common_bsp.c.obj -MF q2pro.exe.p/src_common_bsp.c.obj.d -o q2pro.exe.p/src_common_bsp.c.obj -c ../src/common/bsp.c
../src/common/bsp.c:974:56: error: call to undeclared function 'Com_MakePrintable'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  974 |             Com_WPrintf("Bad material \"%s\" in %s\n", Com_MakePrintable(out->material), path);
      |                                                        ^
../src/common/bsp.c:974:56: warning: format specifies type 'char *' but the argument has type 'int' [-Wformat]
  974 |             Com_WPrintf("Bad material \"%s\" in %s\n", Com_MakePrintable(out->material), path);
      |                                         ~~             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                         %d
../inc/shared/shared.h:112:53: note: expanded from macro 'Com_WPrintf'
  112 | #define Com_WPrintf(...) Com_LPrintf(PRINT_WARNING, __VA_ARGS__)
      |                                                     ^~~~~~~~~~~
1 warning and 1 error generated.
ninja: subcommand failed
```

--- a/inc/common/utils.h
+++ b/inc/common/utils.h
@@ -68,9 +68,7 @@ color_index_t Com_ParseColor(const char *s);
 unsigned Com_ParseExtensionString(const char *s, const char *const extnames[]);
 #endif
 
-#if USE_DEBUG
 char *Com_MakePrintable(const char *s);
-#endif
 
 // Some mods actually exploit CS_STATUSBAR to take space up to CS_AIRACCEL
 static inline size_t CS_SIZE(const cs_remap_t *csr, int cs)

--- a/src/common/utils.c
+++ b/src/common/utils.c
@@ -558,8 +558,6 @@ size_t Com_FormatSizeLong(char *dest, size_t destsize, int64_t bytes)
     return Q_scnprintf(dest, destsize, "unknown size");
 }
 
-#if USE_DEBUG
-
 static int get_escape_char(int c)
 {
     switch (c) {
@@ -597,5 +595,3 @@ char *Com_MakePrintable(const char *s)
     *o = 0;
     return buffer;
 }
-
-#endif

